### ISSUE TITLE
Is there anybody who can merge the Pull requests?? 

### ISSUE THREAD

**Issue by k2dp2k** [PlusPlus-ua/ha_tuya_ble#114]
It´s very bad to not really have an up to date Repo....

**Comment by phid78** [comment]
Maybe because if you read first page until the end, you will note that the owner live in Ukraine...

**Comment by k2dp2k** [comment]
Yeah I now that, I hope he is doing well :-) I just wanted to know if there is anybody else with access to this repo...

**Comment by pathob** [comment]
I'm also concerned about the fact that the owner of this project lives in the Ukraine and hope he is okay. But from his contribution activity graph I can also see in the last years that he had the tendency to be very active over the period of a few months and then "disappeared" for a long time again.

There are some substantial contributions by different people in their own forks, such as @markusg1234 @airy10 @Raph22 @carferrer and [others](https://github.com/PlusPlus-ua/ha_tuya_ble/forks).

Anyone interested in putting your efforts together, create an organisation and create a maintained fork there?

**Comment by pathob** [comment]
I've also contacted @PlusPlus-ua via email, let's see...

**Comment by sirloinofbeef** [comment]
Following with interest as I can't seem to find a fork that works with my BLE lights.

**Comment by edwardtfn** [comment]
Any update to this?
@pathob, have you got any reply?

**Comment by Moe555** [comment]
+1!!

I'm dealing with this issue over and over again since I started with HA/Tuya about a year ago and I really appreciate that somebody opened an issue on this, since I thought about doing the same some time ago.

With every other HA update something breaks completely and another manual workaround is necessary, burried in some comment or (also already abandoned) fork..., even if most of the topics are imo actually covered by the open pull requests contained in this repository.
(For sure the changes on the Tuya-side are also adding to the mix - sometimes I even have the feeling they gradually change things "on purpose".)

And in my case the problem is mainly limited to just keep a BLE Fingerbot working.
I don't want to imagine how others feel having a lot of components in place which are dependent on this integration :D

Don't get me wrong, I really appreciate the work everybody puts into this and I understand and I'm very sorry about the "circumstances" the actual maintainer @PlusPlus-ua is probably suffering at the moment, but this whole thing is just a mess and desperately needs some one taking care of it.

I would really like to somehow support this, I'm even an IT guy, but I'm such a bad developer and never got into git, so I'm not able to take care of this unfortunately...

Is there really nobody able to step into this??
Mabye someone maintaining the several forks?
Or is this some kind of "authorization issue" as well, so that technically nobody can actually do it??
(Sorry for the stupid questions, but I'm, really a hopeless permanent git noob :D)

BR
Moe

**Comment by edwardtfn** [comment]
Well, this project is using MIT license, so anyone can fork, copy or do almost whatever they want, which should include creating a new repo and sharing a copy of this. Convince HACS people to publish migh be another issue.
However, I believe no one have done this before in respect to the situation of this project owner who may be in a difficult situation and incapable of maintaining it.

One approach could be creating a new repo with a promise to transfer it to @PlusPlus-ua when he is back (hopefully soon). There's no guarantees that he will be happy with this approach, but I cannot think in anything more right now.

I can definitely help creating the new repo and trying to convince HACS to publish that (although it still can be used without they publishing it), but I'm definitely not available to invest time on developing or maintaining, other than approving PRs and giving permissions to others to manage it.

**Comment by CloCkWeRX** [comment]
For now, I'm happy to merge all of the various forks into one / make an organisation.

**Comment by CloCkWeRX** [comment]
https://github.com/ha-tuya-ble/ha_tuya_ble created; invited the people mentioned to be owners if they like.

**Comment by carferrer** [comment]
> For now, I'm happy to merge all of the various forks into one / make an organisation.

Great!!
